docs/wiki/dev/core-development.md

```diff
@@ -1,10 +1,91 @@
-# Core development 
+# Life of a Core Developer
+
+![Space Core Devs](../../images/space-core-devs.png)
+
+[Hsiao-Wei Wang](https://github.com/hwwhww) shared the graphic above in a [presentation](https://www.youtube.com/watch?v=0lBrd2_fPPU) she gave at ETHGlobal Tokyo in April 2023. The presentation is titled “A Journey of an Ethereum Core Dev/Researcher” If you have seen this graphic in other presentations, please share!
+
+The graphic symbolizes the connection between a space station and the efforts involved from various teams to bring it to life. You may have also heard Ethereum called the Infinite Garden. With respect to the graphic above, the Infinite Garden could be referred to as space.  
+
+This page will provide an overview of each of the teams/contributors that make the Ethereum Protocol run and what is involved to keep the space station humming. 
+
+One of the most important themes to understand about the different teams working together is that they are all working amongst themselves and amongst eachother in a very decentralized manner. Progress is achieved through discussions carried out in weekly meetings (Exeuction and Consensus Layer calls, EIPIP calls and various "Breakout Room" calls. Outside of calls, discussions are held in the Ethereum R&D Discord channel. Core Devs are writing EIPs, reviewing comments, debating issues/roadmap/frameworks/etc, testing BAU and getting ready for next big releases.  
+
+This page will be split up into several categories: Ethereum Foundation (EF) teams, Execution Layer (EL) client teams, Consensus Layer (CL) client teams and other independent teams/contributors. 
+
+**Ethereum Foundation (EF) teams:**
```

Comment:
I am not sure if we want to maintain a whole list here. Or it could be a table at least. 

A similar list of core teams can be found in PG membership https://protocol-guild.readthedocs.io/en/latest/9-membership.html
This is updated quarterly so maybe the wiki could just refer to that